Dockerfile

```diff
@@ -82,16 +122,27 @@ FROM qtbase as splash-base
 COPY dockerfiles/splash/install-system-splash-deps.sh /tmp/install-system-splash-deps.sh
 RUN /tmp/install-system-splash-deps.sh
 
-# XXX: this needs to be updated if Qt is updated
-COPY --from=qtbuilder /opt/qt-5.13/5.13.1/gcc_64 /opt/qt-5.13/5.13.1/gcc_64
-#RUN ls -l /opt/qt-5.13/5.13.0/gcc_64/lib
-ENV PATH="/opt/qt-5.13/5.13.1/gcc_64/bin:${PATH}"
-ENV LD_LIBRARY_PATH="/opt/qt-5.13/5.13.1/gcc_64/lib:$LD_LIBRARY_PATH"
+ARG QT_SHORT_VERSION
+ARG QT_FULL_VERSION
+
+COPY --from=qtbuilder \
+    /opt/qt-${QT_SHORT_VERSION}/${QT_FULL_VERSION}/gcc_64 \
+    /opt/qt-${QT_SHORT_VERSION}/${QT_FULL_VERSION}/gcc_64
+
+#RUN ls -l /opt/qt-${QT_SHORT_VERSION}/${QT_FULL_VERSION}/gcc_64/lib
+ENV PATH="/opt/qt-${QT_SHORT_VERSION}/${QT_FULL_VERSION}/gcc_64/bin:${PATH}"
+ENV LD_LIBRARY_PATH="/opt/qt-${QT_SHORT_VERSION}/${QT_FULL_VERSION}/gcc_64/lib:$LD_LIBRARY_PATH"
 
 # =====================
 
 FROM splash-base as qt5-builder
 
+ARG QT_MAJOR_VERSION
+ENV QT_MAJOR_VERSION=${QT_MAJOR_VERSION}
```

Comment:
I haven't checked it, but it seems these args are not needed if QT_MAJOR_VERSION is not used to generate PyQt5 package name